addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/models/v2/request/HeatSetPoint.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.evohome.internal.api.models.v2.request;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Request model for the mode
+ *
+ * @author Jasper van Zuijlen - Initial contribution
+ *
+ */
+public class HeatSetPoint {
+
+    /**
+     * Constructs a override reset
+     */
+    public HeatSetPoint() {
+        heatSetpointValue = 0.0;
+        setpointMode = "FollowSchedule";
+        timeUntil = null;
+    }
+
+    /**
+     * Constructs a permanent override with the given temperature
+     *
+     * @param setPoint The target temperature to set the set point to
+     */
+    public HeatSetPoint(double setPoint) {
```

Comment:
I personally would prefer (named) factory methods over this constructor, because now its not completely clear what will happen when you call which constructor, when using the factory method you have the name to clear that up.

Reply:
I don't think a factory makes the design better, I chose to create a builder instead. I'll update the PR tomorrow if I can.